content/docs/2.1/scalers/openstack-swift.md

```diff
@@ -0,0 +1,155 @@
++++
+title = "OpenStack Swift"
+layout = "scaler"
+availability = "v2.1+"
+maintainer = "Community"
+description = "Scale applications based on the count of objects in a given OpenStack Swift container."
+go_file = "openstack_swift_scaler"
++++
+
+### Trigger Specification
+
+This specification describes the `swift` trigger for OpenStack Swift containers. It scales based on the count of objects in a given container.
+
+```yaml
+triggers:
+- type: swift
```

Comment:
Not sure about the name of the scaler,  maybe`openstack-swift` is more appropriate?

Reply:
Makes sense since other scalers use the same standard (azure-queue, azure-blob, redis-streams).

Reply:
Yeah, thanks. Could you please do the same change in the code?

Reply:
Sure! I'm on it 😄

Reply:
Scaler name in documentation updated by 2f61e83.